x-pack/platform/plugins/shared/streams/server/lib/streams/data_streams/manage_data_streams.ts

```diff
@@ -146,6 +147,9 @@ export async function updateDataStreamsLifecycle({
       );
     }
   } catch (err: any) {
+    if (err instanceof errors.ResponseError && err.statusCode === 404) {
```

Comment:
I guess I put this in? But not sure if this is the right place to catch this error. I don't think this PR needs it, so maybe we can remove it?

Reply:
I don't recall adding this. I'll remove